src/src/AdminBundle/Entity/Registration.php

```diff
@@ -151,6 +151,16 @@ public function setConfirmationToken(string $confirmationToken)
         return $this;
     }
 
+    /**
+     * Generate random confirmation token.
+     *
+     * @return self
+     */
+    public function generateConfirmationToken()
+    {
+        return $this->setConfirmationToken(md5(time().rand()));
```

Comment:
md5 je bezpecna? ci to sa neriesi, ked to potrebujes iba na generovanie tokenu?

Reply:
Na heslá to OK nieje tu mi je to jedno